src/pages/plotly.js-basic-dist.d.ts

```diff
@@ -0,0 +1 @@
+declare module "plotly.js-basic-dist";
```

Comment:
why is this diff here? I forget if this is necessary, but if we actually do need it, it should be in a separate pr

Reply:
I was having trouble getting `npm start` to work but it seems to be all good now without this. so I deleted it